js/packages/stream/src/processStreamedMessage.ts

```diff
@@ -0,0 +1,115 @@
+import { AssistantMessage } from "@crayonai/react-core/src/types";
```

Comment:
this is a little problematic.
Depending on react-core would mean depending on react which would mean that backend repos will now start depending on react